apps/agreement/test/helpers/assert/assertBn.js

```diff
@@ -0,0 +1,7 @@
+const assertBn = (actual, expected, errorMsg) => {
```

Comment:
Could all of these helpers be used from or moved to the contract-test-helpers too?

Reply:
Yes, definitely, many of these should be moved to the helpers repo